core/templates/dev/head/pages/skill_editor/skill_editor.html

```diff
@@ -112,16 +113,18 @@
 
   <script src="/templates/dev/head/components/QuestionCreationService.js"></script>
   <script src="/templates/dev/head/components/HintAndSolutionButtonsDirective.js"></script>
-  <script src="/templates/dev/head/components/state/AnswerGroupEditorDirective.js"></script>
-  <script src="/templates/dev/head/components/state/HintEditorDirective.js"></script>
-  <script src="/templates/dev/head/components/state/OutcomeEditorDirective.js"></script>
-  <script src="/templates/dev/head/components/state/OutcomeDestinationEditorDirective.js"></script>
-  <script src="/templates/dev/head/components/state/OutcomeFeedbackEditorDirective.js"></script>
-  <script src="/templates/dev/head/components/state/ResponseHeaderDirective.js"></script>
-  <script src="/templates/dev/head/components/state/RuleEditorDirective.js"></script>
-  <script src="/templates/dev/head/components/state/RuleTypeSelectorDirective.js"></script>
-  <script src="/templates/dev/head/components/state/SolutionEditorDirective.js"></script>
-  <script src="/templates/dev/head/components/state/SolutionExplanationEditorDirective.js"></script>
+  <script src="/templates/dev/head/components/CkEditorRteDirective.js"></script>
+  <script src="/templates/dev/head/components/CkEditorWidgetsInitializer.js"></script>
+  <script src="/templates/dev/head/components/AnswerGroupEditorDirective.js"></script>
+  <script src="/templates/dev/head/components/HintEditorDirective.js"></script>
+  <script src="/templates/dev/head/components/OutcomeEditorDirective.js"></script>
+  <script src="/templates/dev/head/components/OutcomeDestinationEditorDirective.js"></script>
+  <script src="/templates/dev/head/components/OutcomeFeedbackEditorDirective.js"></script>
+  <script src="/templates/dev/head/components/ResponseHeaderDirective.js"></script>
+  <script src="/templates/dev/head/components/RuleEditorDirective.js"></script>
+  <script src="/templates/dev/head/components/RuleTypeSelectorDirective.js"></script>
+  <script src="/templates/dev/head/components/SolutionEditorDirective.js"></script>
+  <script src="/templates/dev/head/components/SolutionExplanationEditorDirective.js"></script>
```

Comment:
You replaced ´/components/state/´ with just ´/components/´, please discard the changes and be more careful next time. Also, don't forget to test everything manually.